common/djangoapps/student/signals/receivers.py

```diff
@@ -58,8 +58,8 @@ def create_course_enrollment_celebration(sender, instance, created, **kwargs):
         return
 
     # The UI for celebrations is only supported on the MFE right now, so don't turn on
-    # celebrations unless this enrollment's course is MFE-enabled.
-    if not REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(instance.course_id):
+    # celebrations unless this enrollment's course is MFE-enabled and has milestones enabled.
+    if not courseware_mfe_first_section_celebration_is_active(instance.course_id):
```

Comment:
we'll need to create the flags in django admin before this merges, so that new enrollments continue to be enabled for the course section celebratory modal